### ISSUE TITLE
Component: Micro Footer - Select Geographic area back button not identified on tabbing.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#3819]
## Detailed description

Issue: With tab/shift-tab focus is not going to the mentioned element.

Expected : Focus should go to each and every interactive element and it should be operable through keyboard.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/e42abc6d-14e5-4aec-8ab9-42d2d4a82a2b)
